common/build.gradle.kts

```diff
@@ -6,9 +6,13 @@ architectury {
 dependencies {
     val fabricLoaderVersion: String by project
     val forgeConfigAPIVersion: String by project
+    val nightConfig: String by project
     // We depend on fabric loader here to use the fabric @Environment annotations and get the mixin dependencies
     // Do NOT use other classes from fabric loader
     modImplementation("net.fabricmc:fabric-loader:${fabricLoaderVersion}")
 
-    api("fuzs.forgeconfigapiport:forgeconfigapiport-common:$forgeConfigAPIVersion")
+    //api("fuzs.forgeconfigapiport:forgeconfigapiport-common:$forgeConfigAPIVersion")
+    api("com.electronwill.night-config:core:$nightConfig")
+    api("com.electronwill.night-config:toml:$nightConfig")
+    api("net.minecraftforge:forgeconfigapiport-fabric:$forgeConfigAPIVersion")
```

Comment:
Why the fabric version on the common project?

Reply:
There's no 1.18.2 common lib iirc, it's just there for the classes, shouldn't effect Forge...

Reply:
fair enough